messages/nl-nl.json

```diff
@@ -1,15 +1,30 @@
 {
 	"title": {
 		"index": "btc.us",
-		"terms-privacy": "Voorwaarden & Privacy",
 		"get": "Claim {domain_name}",
-		"manage": "Mijn domeinen",
+		"manage": "Beheer", //or "Mijn Domain" (singular)
+		"success": "Aankoop succesvol",
+		"web_bridge": "Web Bridge", //soortnaam, kan ook zo?
```

Comment:
Like here.

Reply:
Ah thanks, I have taken those out. It should be better now.